plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/AppEngineJreWhitelistTest.java

```diff
@@ -14,11 +14,13 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.eclipse.appengine.whitelist;
+package com.google.cloud.tools.eclipse.appengine.validation;
 
 import org.junit.Assert;
 import org.junit.Test;
 
+import com.google.cloud.tools.eclipse.appengine.validation.AppEngineJreWhitelist;
```

Comment:
I think this will always happen: when you rename the bundle, the test bundle references will need to import the classes from there. When you then rename the test bundle, Eclipse won't remove them.